pkg/resources/testdata/TestAcc_GrantApplicationRole/application/test.tf

```diff
@@ -0,0 +1,52 @@
+resource "snowflake_stage" "stage" {
+  name     = var.random_name
+  database = var.database_name
+  schema   = var.schema_name
+}
+
+locals {
+  stage_identifier            = "\"${var.database_name}\".\"${var.schema_name}\".\"${snowflake_stage.stage.name}\""
+  file_prefix                 = abspath(path.root)
+  application_role_identifier = "\"${var.application_name}\".\"app_role_1\""
+}
+
+resource "snowflake_unsafe_execute" "put_manifest" {
```

Comment:
these were done before it was easily accessible from helpers, so we can set it up without unsafe execute calls

Reply:
Done.